scanners/gitleaks/helm2.Chart.yaml

```diff
@@ -4,8 +4,8 @@ description: A Helm chart for the gitleaks repository scanner that integrates wi
 
 type: application
 # version - gets automatically set to the secureCodeBox release version when the helm charts gets published
-version: v2.5.0-alpha1
-appVersion: v7.3.0
+version: v2.6.0-alpha1
+appVersion: v6.1.2
```

Comment:
Was it intended to downgrade appVersion and version?

Reply:
nope 🤔